NTPClient.cpp

```diff
@@ -145,6 +145,23 @@ int NTPClient::getSeconds() const {
   return (this->getEpochTime() % 60);
 }
 
+DateTime NTPClient::getDateTime() const {
+  struct tm * ts;
+  time_t rawTime = this->getEpochTime();
+  ts = localtime(&rawTime);
+  DateTime dt = {ts->tm_sec, ts->tm_min, ts->tm_hour, ts->tm_mday, (ts->tm_mon + 1), (ts->tm_year + 1900)};
+  return dt;
+}
+
+String NTPClient::formatDateTime(const char* fmt) const {
```

Comment:
```suggestion
String NTPClient::getFormattedDateTime(const char* fmt) const {
```
This seems more consistent with the existing API, and perhaps what you intended since you called the function `getFormattedDateTime` in the PR description.

Reply:
Updated.

Reply:
Thanks! I like the more descriptive parameter name too.